runtime/src/main/java/io/quarkiverse/logging/cloudwatch/LoggingCloudWatchHandlerValueFactory.java

```diff
@@ -42,9 +45,20 @@ public RuntimeValue<Optional<Handler>> create(final LoggingCloudWatchConfig conf
             return new RuntimeValue<>(Optional.empty());
         }
 
-        config.validate();
-
         LOGGER.info("Initializing Quarkus Logging Cloudwatch Extension");
+
+        List<String> errors = config.validate();
+        if (!errors.isEmpty()) {
+            String errorMsg = "The Quarkus Logging Cloudwatch extension is unable to start because of missing configuration values: "
+                    + String.join(", ", errors);
+            if (ProfileManager.getLaunchMode() == DEVELOPMENT) {
+                LOGGER.error(errorMsg);
+                return new RuntimeValue<>(Optional.empty());
+            } else {
+                throw new IllegalStateException(errorMsg);
+            }
+        }
```

Comment:
@maxandersen Is this what you meant in #361?

Reply:
I'm not sure the hard fail is okay in prod mode but you only mentioned changing the behavior is dev mode IIUC.